types/index.d.ts

```diff
@@ -981,9 +993,9 @@ declare module "jspdf" {
     table(
       x: number,
       y: number,
-      data: any,
-      headers: string[],
-      config: any
+      data: { [key: string]: string }[],
```

Comment:
I think we should also allow numbers and booleans as values here. AFAIK, they work, too.

Reply:
I think numbers and booleans don't work. At the beginning I used numbers and I had to convert all numbers to strings otherwise the fields were empty and the cells were not even drawn. I currently checked it with the unit tests, too. If I use numbers the cells are not drawn. If I use booleans I get the error message "getTextDimensions expects text-parameter to be of type String or type Number or an Array of Strings."

Reply:
Alright. Thanks for checking. I just remembered that numbers are planned to be fixed in #2875. Then I think we can leave it as is.